.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestBase.cs

```diff
@@ -195,7 +196,7 @@ public void TestDateTimeExtractor(TestModel testSpec)
 
             var referenceDateTime = testSpec.GetReferenceDateTime();
 
-            var actualResults = Extractor.Extract(testSpec.Input.ToLowerInvariant(), referenceDateTime);
+            var actualResults = Extractor.Extract(QueryProcessor.Preprocess(testSpec.Input), referenceDateTime);

```

Comment:
Shouldn't this preprocess should happen inside Extract, like in English?

Reply:
what do you mean? Normally this preprocess happens in the Parse method of the models (e.g. in DateTimeModel.cs). But when testing the sub-entities the model is not used.

Reply:
I see. My bad. Forgot this are the internal extractors by sub-type.